include/keyence/impl/high_speed_defs.h

```diff
@@ -3,9 +3,28 @@
 
 #include <stdint.h>
 
+#define KEYENCE_FUNDAMENTAL_LENGTH_UNIT (0.01 * 1.0e-6)
+// The integer encoded values representing 'out of range'
+// received from the sensor
+#define KEYENCE_INFINITE_DISTANCE_VALUE -524288
```

Comment:
There are actually 3 values, but during initial testing / reversing I only saw two. See Section 9.2.9.5 *Profile data values* of the *LJ-V7000 Series Communication Library - Reference Manual*.